plugins/modules/storage/netapp/na_cdot_aggregate.py

```diff
@@ -18,7 +18,7 @@
 
 short_description: Manage NetApp cDOT aggregates.
 extends_documentation_fragment:
-- netapp.ontap.netapp.na_ontap
```

Comment:
I believe the original is correct 
I need to get ` plugins/modules/storage/netapp` moved out of community.general and into https://github.com/ansible-collections/netapp

I think they ended up in community.general when there was confusion around deprecated content and if/how/when it would end up in collections.

Reply:
That was not the original value, but the one I changed it to in #41.